tasks/copy_to_glacier/schemas/output.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/copy_to_files/schemas/output.json",
+  "title": "copy_to_files Lambda Output",
```

Comment:
Shouldn't this be `copy_to_glacier` in the **id**, **title**, and **description**?

Reply:
Whoops. Fixed in all files.